docs/source/user_manual/search_params.rst

```diff
@@ -39,6 +39,11 @@ This document serves to provide a quick overview of the existing parameters and
 |                        |                             | Must be one of ``all``, ``position``,  |
 |                        |                             | or ``mid_position``.                   |
 +------------------------+-----------------------------+----------------------------------------+
+| ``coadds``             | []                          | A list of additional coadds to create. |
+|                        |                             | These are not used in filtering, but   |
+|                        |                             | saved to columns for analysis. Can     |
+|                        |                             | include: "sum", "mean", and "median".  |
++------------------------+-----------------------------+----------------------------------------+
```

Comment:
maybe specify that these are for stamp coadds specifically? my initial thought would be that this would be for a coadd somewhere in the actual kbmod search algorithm and I was slightly confused about it at first 😭

Reply:
You can set the coadd used in the main algorithm using the parameter `stamp_type`. I expanded the comment to clarify this.